Automattic/jetpack
# Social: Add the showNudge reducer

The reducer for showNudge was missing, so it would default back to `false` if a value in the store changed. This meant that after clicking 'Start for Free' on the pricing page, it would seem like the user had a paid plan.

#### Changes proposed in this Pull Request:
This PR adds the missing resolver.

#### Other information:


#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Set up a Jetpack Social site and enable sharing limits
* Click the 'Start for Free' option on the pricing page after connecting the site.
* With this change you'll see the shares meter for the free plan. Without the change you'll see the stats cards, that we show for the paid plan, or if the limits aren't enabled.
